en/asgardeo/docs/guides/monitoring/asgardeo-events.md

```diff
@@ -128,17 +128,19 @@ Follow the steps below to create and deploy a webhook in Choreo.
 
     ![Create a Webhook in Choreo]({{base_path}}/assets/img/guides/asgardeo-events/asgardeo-events-create-webhook-in-choreo.png){: style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
 
-4. Enter a name and a description for your webhook.
+5. Click **Authorize with Github** and connect the relevant organization, repository and the branch of the GitHub repository you created in the above section.
```

Comment:
Shall we point to the choreo docs on deploying a webhook, otherwise asgardeo docs will be outdated every time the choreo console UX is changed?

Reply:
+1. Webhook detailed instructions are now removed from the Asgardeo doc. It points to the Choreo official doc.